src/crypto/local.js

```diff
@@ -270,27 +313,27 @@ function verifyProof(proofObj, tokens, signatures, commitments, prngName) {
     if (chkM.length !== chkZ.length) {
         return false;
     }
-    const pointG = sec1DecodePoint(commitments.G);
-    const pointH = sec1DecodePoint(commitments.H);
+    const pointG = sec1DecodeFromBase64(commitments.G);
+    const pointH = sec1DecodeFromBase64(commitments.H);
 
     // Recompute A and B for proof verification
     const cH = _scalarMult(dleq.C, pointH);
     const rG = _scalarMult(dleq.R, pointG);
     const A = cH.toJac().add(rG).toAffine();
 
-    const composites = recomputeComposites(chkM, chkZ, pointG, pointH, prngName);
+    const composites = recomputeComposites(chkM, chkZ, pointG, pointH, compressed, prngName);
```

Comment:
nit: Passing this ``compressed`` flag doesn't look perfect. If the point is compressed or not is a property of the point  and hence, implementation of a point should have such property. This way you can avoid changing at least ``recomputeComposites`` function (in general - any function which will be only passing a ``Point``)

I would introduce structure wrapping sjcl.ecc.point and add "compressed" field there.

Reply:
I agree the `compressed` flag is annoying. However, I'm slightly against defining an extra struct here because I'm not sure it will help to make things less ambiguous and I would like to enforce that points are received in exactly the same format for an entire server response. So passing around a compression flag for each point seems like overkill.

The latest change removes the flag from most of the function calls, but the compressed flag is still defined with respect to the entire set of tokens received in the response.